viewer/packages/pointclouds/src/styling/shapes/CompositeShape.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { IShape } from './IShape';
+import { IRawShape, ShapeType } from './IRawShape';
+
+// import * as THREE from 'three';
+import { Vec3, AABB, b3Union, emptyBox3 } from './linalg';
+
+export type RawCompositeShape = {
+  type: ShapeType.Composite;
+  shapes: IRawShape[];
+};
+
+export class CompositeShape implements IShape {
+  private readonly _innerShapes: IShape[];
+
+  constructor(shapes: IShape[]) {
+    this._innerShapes = shapes.slice();
+  }
+
+  computeBoundingBox(): AABB {
+    let newBox = emptyBox3();
+    for (const shape of this._innerShapes) {
+      newBox = b3Union(newBox, shape.computeBoundingBox());
+    }
+    return newBox;
```

Comment:
Personally I would use the reduce method here, up to you though.

Reply:
Removed `computeBoundingBox` for now